test/functional/interface_bitcoin_cli.py

```diff
@@ -27,6 +28,41 @@
 WALLET_NOT_LOADED = 'Requested wallet does not exist or is not loaded'
 WALLET_NOT_SPECIFIED = 'Wallet file not specified'
 
+
+def cli_get_info_string_to_dict(cli_get_info_string):
+    # helper method to convert human readable -getinfo into a dictionary
+    cli_get_info = {}
+    lines = cli_get_info_string.splitlines()
+    line_idx = 0
+    ansi_escape = re.compile(r'(\x9B|\x1B\[)[0-?]*[ -\/]*[@-~]')
+    while line_idx < len(lines):
+        # Remove ansi colour code
+        line = ansi_escape.sub('', lines[line_idx])
+        if "Balances" in line:
+            # When Balances appears in a line, all of the following line contains "balance: wallet" until an empty line
```

Comment:
```suggestion
            # When "Balances" appears in a line, all of the following lines contain "balance: wallet" until an empty line
```

Reply:
Good catch! Resolved it in [`b634ff2`](https://github.com/bitcoin/bitcoin/commit/b634ff26ab8cf4bdffed1a75300ec63a9da22cf1)